# ACNE

<details>

* Version: 0.8.1
* GitHub: https://github.com/HenrikBengtsson/ACNE
* Source code: https://github.com/cran/ACNE
* Date/Publication: 2015-10-27 08:09:51
* Number of recursive dependencies: 25

Run `revdep_details(, "ACNE")` for more info

</details>

## Newly broken

*   checking whether package ‘ACNE’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/c4/home/henrik/repositories/R.utils/revdep/checks/ACNE/new/ACNE.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘ACNE’ ...
** package ‘ACNE’ successfully unpacked and MD5 sums checked
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘R.devices’:
 object ‘inherits’ is not exported by 'namespace:R.utils'
Execution halted
ERROR: lazy loading failed for package ‘ACNE’
* removing ‘/c4/home/henrik/repositories/R.utils/revdep/checks/ACNE/new/ACNE.Rcheck/ACNE’


```
### CRAN

```
* installing *source* package ‘ACNE’ ...
** package ‘ACNE’ successfully unpacked and MD5 sums checked
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
Warning message:
In .requireBiocPackage("affxparser", neededBy = getName(pkg)) :
  Package 'affxparser' could not be loaded. Without it aroma.affymetrix will not work. Please install it from Bioconductor, cf. https://www.bioconductor.org/
** help
...
*** installing help indices
** building package indices
** testing if installed package can be loaded from temporary location
Warning in .requireBiocPackage("affxparser", neededBy = getName(pkg)) :
  Package 'affxparser' could not be loaded. Without it aroma.affymetrix will not work. Please install it from Bioconductor, cf. https://www.bioconductor.org/
** testing if installed package can be loaded from final location
Warning in .requireBiocPackage("affxparser", neededBy = getName(pkg)) :
  Package 'affxparser' could not be loaded. Without it aroma.affymetrix will not work. Please install it from Bioconductor, cf. https://www.bioconductor.org/
** testing if installed package keeps a record of temporary installation path
* DONE (ACNE)


```
# aroma.affymetrix

<details>

* Version: 3.2.0
* GitHub: https://github.com/HenrikBengtsson/aroma.affymetrix
* Source code: https://github.com/cran/aroma.affymetrix
* Date/Publication: 2019-06-23 06:00:14 UTC
* Number of recursive dependencies: 73

Run `revdep_details(, "aroma.affymetrix")` for more info

</details>

## Newly broken

*   checking whether package ‘aroma.affymetrix’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/c4/home/henrik/repositories/R.utils/revdep/checks/aroma.affymetrix/new/aroma.affymetrix.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘aroma.affymetrix’ ...
** package ‘aroma.affymetrix’ successfully unpacked and MD5 sums checked
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘R.devices’:
 object ‘inherits’ is not exported by 'namespace:R.utils'
Execution halted
ERROR: lazy loading failed for package ‘aroma.affymetrix’
* removing ‘/c4/home/henrik/repositories/R.utils/revdep/checks/aroma.affymetrix/new/aroma.affymetrix.Rcheck/aroma.affymetrix’


```
### CRAN

```
* installing *source* package ‘aroma.affymetrix’ ...
** package ‘aroma.affymetrix’ successfully unpacked and MD5 sums checked
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** testing if installed package can be loaded from temporary location
...
The following object is masked _by_ package:aroma.affymetrix:

    writeCdf

The following object is masked from package:R.utils:

    findFiles

** testing if installed package keeps a record of temporary installation path
* DONE (aroma.affymetrix)


```
# aroma.cn

<details>

* Version: 1.6.1
* GitHub: https://github.com/HenrikBengtsson/aroma.cn
* Source code: https://github.com/cran/aroma.cn
* Date/Publication: 2015-10-28 00:08:16
* Number of recursive dependencies: 25

Run `revdep_details(, "aroma.cn")` for more info

</details>

## Newly broken

*   checking whether package ‘aroma.cn’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/c4/home/henrik/repositories/R.utils/revdep/checks/aroma.cn/new/aroma.cn.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘aroma.cn’ ...
** package ‘aroma.cn’ successfully unpacked and MD5 sums checked
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘R.devices’:
 object ‘inherits’ is not exported by 'namespace:R.utils'
Execution halted
ERROR: lazy loading failed for package ‘aroma.cn’
* removing ‘/c4/home/henrik/repositories/R.utils/revdep/checks/aroma.cn/new/aroma.cn.Rcheck/aroma.cn’


```
### CRAN

```
* installing *source* package ‘aroma.cn’ ...
** package ‘aroma.cn’ successfully unpacked and MD5 sums checked
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (aroma.cn)


```
# aroma.core

<details>

* Version: 3.2.2
* GitHub: https://github.com/HenrikBengtsson/aroma.core
* Source code: https://github.com/cran/aroma.core
* Date/Publication: 2021-01-05 05:10:12 UTC
* Number of recursive dependencies: 48

Run `revdep_details(, "aroma.core")` for more info

</details>

## Newly broken

*   checking whether package ‘aroma.core’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/c4/home/henrik/repositories/R.utils/revdep/checks/aroma.core/new/aroma.core.Rcheck/00install.out’ for details.
    ```

## In both

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      'sfit', 'expectile', 'HaarSeg', 'mpcbs'
    ```

## Installation

### Devel

```
* installing *source* package ‘aroma.core’ ...
** package ‘aroma.core’ successfully unpacked and MD5 sums checked
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘R.devices’:
 object ‘inherits’ is not exported by 'namespace:R.utils'
Execution halted
ERROR: lazy loading failed for package ‘aroma.core’
* removing ‘/c4/home/henrik/repositories/R.utils/revdep/checks/aroma.core/new/aroma.core.Rcheck/aroma.core’


```
### CRAN

```
* installing *source* package ‘aroma.core’ ...
** package ‘aroma.core’ successfully unpacked and MD5 sums checked
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
Warning in setGenericS3.default(name, export = exportGeneric, envir = envir,  :
  Renamed the preexisting function write to write.default, which was defined in environment base.
** help
*** installing help indices
** building package indices
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (aroma.core)


```
# BgeeDB

<details>

* Version: 2.22.0
* GitHub: https://github.com/BgeeDB/BgeeDB_R
* Source code: https://github.com/cran/BgeeDB
* Date/Publication: 2022-04-26
* Number of recursive dependencies: 96

Run `revdep_details(, "BgeeDB")` for more info

</details>

## In both

*   R CMD check timed out
    

# biscuiteer

<details>

* Version: 
* GitHub: https://github.com/HenrikBengtsson/R.utils
* Source code: NA
* Number of recursive dependencies: 0

</details>

## Error before installation

### Devel

```
Creating a new generic function for ‘append’ in package ‘BiocGenerics’
Creating a new generic function for ‘as.data.frame’ in package ‘BiocGenerics’
Creating a new generic function for ‘cbind’ in package ‘BiocGenerics’
Creating a new generic function for ‘rbind’ in package ‘BiocGenerics’
Creating a new generic function for ‘do.call’ in package ‘BiocGenerics’
Creating a new generic function for ‘duplicated’ in package ‘BiocGenerics’
Creating a new generic function for ‘anyDuplicated’ in package ‘BiocGenerics’
Creating a new generic function for ‘eval’ in package ‘BiocGenerics’
Creating a new generic function for ‘pmax’ in package ‘BiocGenerics’
Creating a new generic function for ‘pmin’ in package ‘BiocGenerics’
...
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -D_FILE_OFFSET_BITS=64 -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/S4Vectors/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/IRanges/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/XVector/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/Biostrings/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/Rhtslib/include' -I/usr/local/include   -fpic  -g -O2  -c XVector_stubs.c -o XVector_stubs.o
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -D_FILE_OFFSET_BITS=64 -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/S4Vectors/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/IRanges/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/XVector/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/Biostrings/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/Rhtslib/include' -I/usr/local/include   -fpic  -g -O2  -c dna_hash.c -o dna_hash.o
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -D_FILE_OFFSET_BITS=64 -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/S4Vectors/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/IRanges/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/XVector/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/Biostrings/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/Rhtslib/include' -I/usr/local/include   -fpic  -g -O2  -c rle.c -o rle.o
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -D_FILE_OFFSET_BITS=64 -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/S4Vectors/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/IRanges/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/XVector/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/Biostrings/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/Rhtslib/include' -I/usr/local/include   -fpic  -g -O2  -c strhash.c -o strhash.o
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -D_FILE_OFFSET_BITS=64 -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/S4Vectors/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/IRanges/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/XVector/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/Biostrings/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/Rhtslib/include' -I/usr/local/include   -fpic  -g -O2  -c utilities.c -o utilities.o
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -D_FILE_OFFSET_BITS=64 -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/S4Vectors/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/IRanges/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/XVector/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/Biostrings/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/Rhtslib/include' -I/usr/local/include   -fpic  -g -O2  -c vcffile.c -o vcffile.o
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -D_FILE_OFFSET_BITS=64 -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/S4Vectors/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/IRanges/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/XVector/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/Biostrings/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/Rhtslib/include' -I/usr/local/include   -fpic  -g -O2  -c vcftype.c -o vcftype.o
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -D_FILE_OFFSET_BITS=64 -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/S4Vectors/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/IRanges/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/XVector/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/Biostrings/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/Rhtslib/include' -I/usr/local/include   -fpic  -g -O2  -c writevcf.c -o writevcf.o
gcc -shared -L/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/lib -L/usr/local/lib64 -o VariantAnnotation.so Biostrings_stubs.o IRanges_stubs.o R_init_VariantAnnotation.o XVector_stubs.o dna_hash.o rle.o strhash.o utilities.o vcffile.o vcftype.o writevcf.o /c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/Rhtslib/usrlib/libhts.a -lcurl -L/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/lib -lR
Creating a new generic function for ‘tabulate’ in package ‘VariantAnnotation’


trying URL 'https://bioconductor.org/packages/3.15/data/experiment/src/contrib/biscuiteerData_1.10.0.tar.gz'
Content type 'application/x-gzip' length 254116 bytes (248 KB)
==================================================
downloaded 248 KB

trying URL 'https://bioconductor.org/packages/3.15/bioc/src/contrib/bsseq_1.32.0.tar.gz'
Content type 'application/x-gzip' length 2856504 bytes (2.7 MB)
==================================================
downloaded 2.7 MB

...
* DONE (dmrseq)
* installing *binary* package ‘Homo.sapiens’ ...
* DONE (Homo.sapiens)
* installing *binary* package ‘Mus.musculus’ ...
* DONE (Mus.musculus)
* installing *binary* package ‘scmeth’ ...
* DONE (scmeth)

The downloaded source packages are in
	‘/scratch/henrik/641569/RtmptQH0mg/downloaded_packages’


```
### CRAN

```
Creating a new generic function for ‘append’ in package ‘BiocGenerics’
Creating a new generic function for ‘as.data.frame’ in package ‘BiocGenerics’
Creating a new generic function for ‘cbind’ in package ‘BiocGenerics’
Creating a new generic function for ‘rbind’ in package ‘BiocGenerics’
Creating a new generic function for ‘do.call’ in package ‘BiocGenerics’
Creating a new generic function for ‘duplicated’ in package ‘BiocGenerics’
Creating a new generic function for ‘anyDuplicated’ in package ‘BiocGenerics’
Creating a new generic function for ‘eval’ in package ‘BiocGenerics’
Creating a new generic function for ‘pmax’ in package ‘BiocGenerics’
Creating a new generic function for ‘pmin’ in package ‘BiocGenerics’
...
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -D_FILE_OFFSET_BITS=64 -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/S4Vectors/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/IRanges/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/XVector/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/Biostrings/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/Rhtslib/include' -I/usr/local/include   -fpic  -g -O2  -c XVector_stubs.c -o XVector_stubs.o
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -D_FILE_OFFSET_BITS=64 -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/S4Vectors/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/IRanges/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/XVector/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/Biostrings/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/Rhtslib/include' -I/usr/local/include   -fpic  -g -O2  -c dna_hash.c -o dna_hash.o
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -D_FILE_OFFSET_BITS=64 -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/S4Vectors/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/IRanges/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/XVector/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/Biostrings/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/Rhtslib/include' -I/usr/local/include   -fpic  -g -O2  -c rle.c -o rle.o
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -D_FILE_OFFSET_BITS=64 -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/S4Vectors/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/IRanges/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/XVector/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/Biostrings/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/Rhtslib/include' -I/usr/local/include   -fpic  -g -O2  -c strhash.c -o strhash.o
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -D_FILE_OFFSET_BITS=64 -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/S4Vectors/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/IRanges/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/XVector/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/Biostrings/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/Rhtslib/include' -I/usr/local/include   -fpic  -g -O2  -c utilities.c -o utilities.o
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -D_FILE_OFFSET_BITS=64 -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/S4Vectors/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/IRanges/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/XVector/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/Biostrings/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/Rhtslib/include' -I/usr/local/include   -fpic  -g -O2  -c vcffile.c -o vcffile.o
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -D_FILE_OFFSET_BITS=64 -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/S4Vectors/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/IRanges/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/XVector/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/Biostrings/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/Rhtslib/include' -I/usr/local/include   -fpic  -g -O2  -c vcftype.c -o vcftype.o
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -D_FILE_OFFSET_BITS=64 -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/S4Vectors/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/IRanges/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/XVector/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/Biostrings/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/Rhtslib/include' -I/usr/local/include   -fpic  -g -O2  -c writevcf.c -o writevcf.o
gcc -shared -L/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/lib -L/usr/local/lib64 -o VariantAnnotation.so Biostrings_stubs.o IRanges_stubs.o R_init_VariantAnnotation.o XVector_stubs.o dna_hash.o rle.o strhash.o utilities.o vcffile.o vcftype.o writevcf.o /c4/home/henrik/repositories/R.utils/revdep/library/biscuiteer/Rhtslib/usrlib/libhts.a -lcurl -L/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/lib -lR
Creating a new generic function for ‘tabulate’ in package ‘VariantAnnotation’


trying URL 'https://bioconductor.org/packages/3.15/data/experiment/src/contrib/biscuiteerData_1.10.0.tar.gz'
Content type 'application/x-gzip' length 254116 bytes (248 KB)
==================================================
downloaded 248 KB

trying URL 'https://bioconductor.org/packages/3.15/bioc/src/contrib/bsseq_1.32.0.tar.gz'
Content type 'application/x-gzip' length 2856504 bytes (2.7 MB)
==================================================
downloaded 2.7 MB

...
* DONE (dmrseq)
* installing *binary* package ‘Homo.sapiens’ ...
* DONE (Homo.sapiens)
* installing *binary* package ‘Mus.musculus’ ...
* DONE (Mus.musculus)
* installing *binary* package ‘scmeth’ ...
* DONE (scmeth)

The downloaded source packages are in
	‘/scratch/henrik/641569/RtmptQH0mg/downloaded_packages’


```
# calmate

<details>

* Version: 0.13.0
* GitHub: https://github.com/HenrikBengtsson/calmate
* Source code: https://github.com/cran/calmate
* Date/Publication: 2022-03-08 23:00:03 UTC
* Number of recursive dependencies: 22

Run `revdep_details(, "calmate")` for more info

</details>

## Newly broken

*   checking whether package ‘calmate’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/c4/home/henrik/repositories/R.utils/revdep/checks/calmate/new/calmate.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘calmate’ ...
** package ‘calmate’ successfully unpacked and MD5 sums checked
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘R.devices’:
 object ‘inherits’ is not exported by 'namespace:R.utils'
Execution halted
ERROR: lazy loading failed for package ‘calmate’
* removing ‘/c4/home/henrik/repositories/R.utils/revdep/checks/calmate/new/calmate.Rcheck/calmate’


```
### CRAN

```
* installing *source* package ‘calmate’ ...
** package ‘calmate’ successfully unpacked and MD5 sums checked
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (calmate)


```
# CINdex

<details>

* Version: 
* GitHub: https://github.com/HenrikBengtsson/R.utils
* Source code: NA
* Number of recursive dependencies: 0

</details>

## Error before installation

### Devel

```
rm -f fusion/calvin_files/data/src/CDFData.o fusion/calvin_files/data/src/CDFProbeGroupInformation.o fusion/calvin_files/data/src/CDFProbeInformation.o fusion/calvin_files/data/src/CDFProbeSetInformation.o fusion/calvin_files/data/src/CDFQCProbeInformation.o fusion/calvin_files/data/src/CDFQCProbeSetInformation.o fusion/calvin_files/data/src/CELData.o fusion/calvin_files/data/src/CHPBackgroundZone.o fusion/calvin_files/data/src/CHPData.o fusion/calvin_files/data/src/CHPExpressionEntry.o fusion/calvin_files/data/src/CHPMultiDataData.o fusion/calvin_files/data/src/CHPTilingData.o fusion/calvin_files/data/src/CHPQuantificationData.o fusion/calvin_files/data/src/CHPQuantificationDetectionData.o fusion/calvin_files/data/src/CHPGenotypeEntry.o fusion/calvin_files/data/src/CHPUniversalEntry.o fusion/calvin_files/data/src/ColumnInfo.o fusion/calvin_files/data/src/DataGroup.o fusion/calvin_files/data/src/DataGroupHeader.o fusion/calvin_files/data/src/DataSet.o fusion/calvin_files/data/src/DataSetHeader.o fusion/calvin_files/data/src/FileHeader.o fusion/calvin_files/data/src/GenericData.o fusion/calvin_files/data/src/GenericDataHeader.o fusion/calvin_files/exception/src/ExceptionBase.o fusion/calvin_files/fusion/src/CalvinAdapter/CalvinCELDataAdapter.o fusion/calvin_files/fusion/src/CalvinAdapter/CalvinCHPDataAdapter.o fusion/calvin_files/fusion/src/FusionBPMAPData.o fusion/calvin_files/fusion/src/FusionCDFData.o fusion/calvin_files/fusion/src/FusionCDFQCProbeSetNames.o fusion/calvin_files/fusion/src/FusionCELData.o fusion/calvin_files/fusion/src/FusionCHPData.o fusion/calvin_files/fusion/src/FusionProbeSetResults.o fusion/calvin_files/fusion/src/GCOSAdapter/GCOSCELDataAdapter.o fusion/calvin_files/fusion/src/GCOSAdapter/GCOSCHPDataAdapter.o fusion/calvin_files/fusion/src/FusionCHPLegacyData.o fusion/calvin_files/fusion/src/FusionCHPMultiDataAccessor.o fusion/calvin_files/fusion/src/FusionCHPMultiDataData.o fusion/calvin_files/fusion/src/FusionCHPTilingData.o fusion/calvin_files/fusion/src/FusionCHPGenericData.o fusion/calvin_files/fusion/src/FusionCHPQuantificationData.o fusion/calvin_files/fusion/src/FusionCHPQuantificationDetectionData.o fusion/calvin_files/parameter/src/ParameterNameValueType.o fusion/calvin_files/parsers/src/CDFFileReader.o fusion/calvin_files/parsers/src/CelFileReader.o fusion/calvin_files/parsers/src/CHPFileReader.o fusion/calvin_files/parsers/src/CHPMultiDataFileReader.o fusion/calvin_files/parsers/src/CHPTilingFileReader.o fusion/calvin_files/parsers/src/CHPQuantificationFileReader.o fusion/calvin_files/parsers/src/CHPQuantificationDetectionFileReader.o fusion/calvin_files/parsers/src/DataGroupHeaderReader.o fusion/calvin_files/parsers/src/DataGroupReader.o fusion/calvin_files/parsers/src/DataSetHeaderReader.o fusion/calvin_files/parsers/src/DataSetReader.o fusion/calvin_files/parsers/src/FileHeaderReader.o fusion/calvin_files/parsers/src/FileInput.o fusion/calvin_files/parsers/src/GenericDataHeaderReader.o fusion/calvin_files/parsers/src/GenericFileReader.o fusion/calvin_files/utils/src/AffymetrixGuid.o fusion/calvin_files/utils/src/DateTime.o fusion/calvin_files/utils/src/FileUtils.o fusion/calvin_files/utils/src/StringUtils.o fusion/calvin_files/utils/src/checksum.o fusion/file/BPMAPFileData.o fusion/file/BPMAPFileWriter.o fusion/file/CDFFileData.o fusion/file/CELFileData.o fusion/file/CHPFileData.o fusion/file/FileIO.o fusion/file/FileWriter.o fusion/file/TsvFile/ClfFile.o fusion/file/TsvFile/PgfFile.o fusion/file/TsvFile/TsvFile.o fusion/util/AffxByteArray.o fusion/util/AffxConv.o fusion/util/MsgStream.o fusion/util/Util.o fusion/util/Err.o fusion/util/Fs.o fusion/util/Verbose.o fusion/util/RowFile.o fusion/util/TableFile.o fusion/util/Convert.o R_affx_cel_parser.o R_affx_cdf_parser.o R_affx_cdf_extras.o R_affx_bpmap_parser.o R_affx_clf_pgf_parser.o R_affx_chp_parser.o 000.init.o
g++ -std=gnu++14 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -I. -Ifusion/calvin_files/array/src -Ifusion/calvin_files/data/src -Ifusion/calvin_files/exception/src -Ifusion/calvin_files/fusion/src -Ifusion/calvin_files/fusion/src/GCOSAdapter -Ifusion/calvin_files/fusion/src/CalvinAdapter -Ifusion/calvin_files/parameter/src -Ifusion/calvin_files/parsers/src -Ifusion/calvin_files/portability/src -Ifusion/calvin_files/template/src -Ifusion/calvin_files/utils/src -Ifusion/calvin_files/writers/src -Ifusion/file -Ifusion/file/TsvFile -Ifusion/portability -Ifusion/util -Ifusion -D_USE_MEM_MAPPING_  -I/usr/local/include   -fpic  -g -O2  -Wno-sign-compare -O0 -c fusion/calvin_files/data/src/CDFData.cpp -o fusion/calvin_files/data/src/CDFData.o
g++ -std=gnu++14 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -I. -Ifusion/calvin_files/array/src -Ifusion/calvin_files/data/src -Ifusion/calvin_files/exception/src -Ifusion/calvin_files/fusion/src -Ifusion/calvin_files/fusion/src/GCOSAdapter -Ifusion/calvin_files/fusion/src/CalvinAdapter -Ifusion/calvin_files/parameter/src -Ifusion/calvin_files/parsers/src -Ifusion/calvin_files/portability/src -Ifusion/calvin_files/template/src -Ifusion/calvin_files/utils/src -Ifusion/calvin_files/writers/src -Ifusion/file -Ifusion/file/TsvFile -Ifusion/portability -Ifusion/util -Ifusion -D_USE_MEM_MAPPING_  -I/usr/local/include   -fpic  -g -O2  -Wno-sign-compare -O0 -c fusion/calvin_files/data/src/CDFProbeGroupInformation.cpp -o fusion/calvin_files/data/src/CDFProbeGroupInformation.o
g++ -std=gnu++14 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -I. -Ifusion/calvin_files/array/src -Ifusion/calvin_files/data/src -Ifusion/calvin_files/exception/src -Ifusion/calvin_files/fusion/src -Ifusion/calvin_files/fusion/src/GCOSAdapter -Ifusion/calvin_files/fusion/src/CalvinAdapter -Ifusion/calvin_files/parameter/src -Ifusion/calvin_files/parsers/src -Ifusion/calvin_files/portability/src -Ifusion/calvin_files/template/src -Ifusion/calvin_files/utils/src -Ifusion/calvin_files/writers/src -Ifusion/file -Ifusion/file/TsvFile -Ifusion/portability -Ifusion/util -Ifusion -D_USE_MEM_MAPPING_  -I/usr/local/include   -fpic  -g -O2  -Wno-sign-compare -O0 -c fusion/calvin_files/data/src/CDFProbeInformation.cpp -o fusion/calvin_files/data/src/CDFProbeInformation.o
g++ -std=gnu++14 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -I. -Ifusion/calvin_files/array/src -Ifusion/calvin_files/data/src -Ifusion/calvin_files/exception/src -Ifusion/calvin_files/fusion/src -Ifusion/calvin_files/fusion/src/GCOSAdapter -Ifusion/calvin_files/fusion/src/CalvinAdapter -Ifusion/calvin_files/parameter/src -Ifusion/calvin_files/parsers/src -Ifusion/calvin_files/portability/src -Ifusion/calvin_files/template/src -Ifusion/calvin_files/utils/src -Ifusion/calvin_files/writers/src -Ifusion/file -Ifusion/file/TsvFile -Ifusion/portability -Ifusion/util -Ifusion -D_USE_MEM_MAPPING_  -I/usr/local/include   -fpic  -g -O2  -Wno-sign-compare -O0 -c fusion/calvin_files/data/src/CDFProbeSetInformation.cpp -o fusion/calvin_files/data/src/CDFProbeSetInformation.o
g++ -std=gnu++14 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -I. -Ifusion/calvin_files/array/src -Ifusion/calvin_files/data/src -Ifusion/calvin_files/exception/src -Ifusion/calvin_files/fusion/src -Ifusion/calvin_files/fusion/src/GCOSAdapter -Ifusion/calvin_files/fusion/src/CalvinAdapter -Ifusion/calvin_files/parameter/src -Ifusion/calvin_files/parsers/src -Ifusion/calvin_files/portability/src -Ifusion/calvin_files/template/src -Ifusion/calvin_files/utils/src -Ifusion/calvin_files/writers/src -Ifusion/file -Ifusion/file/TsvFile -Ifusion/portability -Ifusion/util -Ifusion -D_USE_MEM_MAPPING_  -I/usr/local/include   -fpic  -g -O2  -Wno-sign-compare -O0 -c fusion/calvin_files/data/src/CDFQCProbeInformation.cpp -o fusion/calvin_files/data/src/CDFQCProbeInformation.o
g++ -std=gnu++14 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -I. -Ifusion/calvin_files/array/src -Ifusion/calvin_files/data/src -Ifusion/calvin_files/exception/src -Ifusion/calvin_files/fusion/src -Ifusion/calvin_files/fusion/src/GCOSAdapter -Ifusion/calvin_files/fusion/src/CalvinAdapter -Ifusion/calvin_files/parameter/src -Ifusion/calvin_files/parsers/src -Ifusion/calvin_files/portability/src -Ifusion/calvin_files/template/src -Ifusion/calvin_files/utils/src -Ifusion/calvin_files/writers/src -Ifusion/file -Ifusion/file/TsvFile -Ifusion/portability -Ifusion/util -Ifusion -D_USE_MEM_MAPPING_  -I/usr/local/include   -fpic  -g -O2  -Wno-sign-compare -O0 -c fusion/calvin_files/data/src/CDFQCProbeSetInformation.cpp -o fusion/calvin_files/data/src/CDFQCProbeSetInformation.o
g++ -std=gnu++14 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -I. -Ifusion/calvin_files/array/src -Ifusion/calvin_files/data/src -Ifusion/calvin_files/exception/src -Ifusion/calvin_files/fusion/src -Ifusion/calvin_files/fusion/src/GCOSAdapter -Ifusion/calvin_files/fusion/src/CalvinAdapter -Ifusion/calvin_files/parameter/src -Ifusion/calvin_files/parsers/src -Ifusion/calvin_files/portability/src -Ifusion/calvin_files/template/src -Ifusion/calvin_files/utils/src -Ifusion/calvin_files/writers/src -Ifusion/file -Ifusion/file/TsvFile -Ifusion/portability -Ifusion/util -Ifusion -D_USE_MEM_MAPPING_  -I/usr/local/include   -fpic  -g -O2  -Wno-sign-compare -O0 -c fusion/calvin_files/data/src/CELData.cpp -o fusion/calvin_files/data/src/CELData.o
g++ -std=gnu++14 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -I. -Ifusion/calvin_files/array/src -Ifusion/calvin_files/data/src -Ifusion/calvin_files/exception/src -Ifusion/calvin_files/fusion/src -Ifusion/calvin_files/fusion/src/GCOSAdapter -Ifusion/calvin_files/fusion/src/CalvinAdapter -Ifusion/calvin_files/parameter/src -Ifusion/calvin_files/parsers/src -Ifusion/calvin_files/portability/src -Ifusion/calvin_files/template/src -Ifusion/calvin_files/utils/src -Ifusion/calvin_files/writers/src -Ifusion/file -Ifusion/file/TsvFile -Ifusion/portability -Ifusion/util -Ifusion -D_USE_MEM_MAPPING_  -I/usr/local/include   -fpic  -g -O2  -Wno-sign-compare -O0 -c fusion/calvin_files/data/src/CHPBackgroundZone.cpp -o fusion/calvin_files/data/src/CHPBackgroundZone.o
g++ -std=gnu++14 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -I. -Ifusion/calvin_files/array/src -Ifusion/calvin_files/data/src -Ifusion/calvin_files/exception/src -Ifusion/calvin_files/fusion/src -Ifusion/calvin_files/fusion/src/GCOSAdapter -Ifusion/calvin_files/fusion/src/CalvinAdapter -Ifusion/calvin_files/parameter/src -Ifusion/calvin_files/parsers/src -Ifusion/calvin_files/portability/src -Ifusion/calvin_files/template/src -Ifusion/calvin_files/utils/src -Ifusion/calvin_files/writers/src -Ifusion/file -Ifusion/file/TsvFile -Ifusion/portability -Ifusion/util -Ifusion -D_USE_MEM_MAPPING_  -I/usr/local/include   -fpic  -g -O2  -Wno-sign-compare -O0 -c fusion/calvin_files/data/src/CHPData.cpp -o fusion/calvin_files/data/src/CHPData.o
...
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -D_FILE_OFFSET_BITS=64 -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/S4Vectors/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/IRanges/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/XVector/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/Biostrings/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/Rhtslib/include' -I/usr/local/include   -fpic  -g -O2  -c strhash.c -o strhash.o
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -D_FILE_OFFSET_BITS=64 -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/S4Vectors/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/IRanges/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/XVector/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/Biostrings/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/Rhtslib/include' -I/usr/local/include   -fpic  -g -O2  -c utilities.c -o utilities.o
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -D_FILE_OFFSET_BITS=64 -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/S4Vectors/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/IRanges/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/XVector/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/Biostrings/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/Rhtslib/include' -I/usr/local/include   -fpic  -g -O2  -c vcffile.c -o vcffile.o
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -D_FILE_OFFSET_BITS=64 -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/S4Vectors/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/IRanges/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/XVector/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/Biostrings/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/Rhtslib/include' -I/usr/local/include   -fpic  -g -O2  -c vcftype.c -o vcftype.o
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -D_FILE_OFFSET_BITS=64 -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/S4Vectors/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/IRanges/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/XVector/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/Biostrings/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/Rhtslib/include' -I/usr/local/include   -fpic  -g -O2  -c writevcf.c -o writevcf.o
gcc -shared -L/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/lib -L/usr/local/lib64 -o VariantAnnotation.so Biostrings_stubs.o IRanges_stubs.o R_init_VariantAnnotation.o XVector_stubs.o dna_hash.o rle.o strhash.o utilities.o vcffile.o vcftype.o writevcf.o /c4/home/henrik/repositories/R.utils/revdep/library/CINdex/Rhtslib/usrlib/libhts.a -lcurl -L/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/lib -lR
Creating a new generic function for ‘tabulate’ in package ‘VariantAnnotation’
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG   -I/usr/local/include   -fpic  -g -O2  -c R_init_biovizBase.c -o R_init_biovizBase.o
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG   -I/usr/local/include   -fpic  -g -O2  -c bin_offsets.c -o bin_offsets.o
gcc -shared -L/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/lib -L/usr/local/lib64 -o biovizBase.so R_init_biovizBase.o bin_offsets.o -L/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/lib -lR


trying URL 'https://bioconductor.org/packages/3.15/bioc/src/contrib/enrichplot_1.16.1.tar.gz'
Content type 'application/x-gzip' length 93082 bytes (90 KB)
==================================================
downloaded 90 KB

trying URL 'https://bioconductor.org/packages/3.15/bioc/src/contrib/graphite_1.42.0.tar.gz'
Content type 'application/x-gzip' length 780805 bytes (762 KB)
==================================================
downloaded 762 KB

...
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (ReactomePA)

The downloaded source packages are in
	‘/scratch/henrik/641569/RtmphNjpta/downloaded_packages’


```
### CRAN

```
rm -f fusion/calvin_files/data/src/CDFData.o fusion/calvin_files/data/src/CDFProbeGroupInformation.o fusion/calvin_files/data/src/CDFProbeInformation.o fusion/calvin_files/data/src/CDFProbeSetInformation.o fusion/calvin_files/data/src/CDFQCProbeInformation.o fusion/calvin_files/data/src/CDFQCProbeSetInformation.o fusion/calvin_files/data/src/CELData.o fusion/calvin_files/data/src/CHPBackgroundZone.o fusion/calvin_files/data/src/CHPData.o fusion/calvin_files/data/src/CHPExpressionEntry.o fusion/calvin_files/data/src/CHPMultiDataData.o fusion/calvin_files/data/src/CHPTilingData.o fusion/calvin_files/data/src/CHPQuantificationData.o fusion/calvin_files/data/src/CHPQuantificationDetectionData.o fusion/calvin_files/data/src/CHPGenotypeEntry.o fusion/calvin_files/data/src/CHPUniversalEntry.o fusion/calvin_files/data/src/ColumnInfo.o fusion/calvin_files/data/src/DataGroup.o fusion/calvin_files/data/src/DataGroupHeader.o fusion/calvin_files/data/src/DataSet.o fusion/calvin_files/data/src/DataSetHeader.o fusion/calvin_files/data/src/FileHeader.o fusion/calvin_files/data/src/GenericData.o fusion/calvin_files/data/src/GenericDataHeader.o fusion/calvin_files/exception/src/ExceptionBase.o fusion/calvin_files/fusion/src/CalvinAdapter/CalvinCELDataAdapter.o fusion/calvin_files/fusion/src/CalvinAdapter/CalvinCHPDataAdapter.o fusion/calvin_files/fusion/src/FusionBPMAPData.o fusion/calvin_files/fusion/src/FusionCDFData.o fusion/calvin_files/fusion/src/FusionCDFQCProbeSetNames.o fusion/calvin_files/fusion/src/FusionCELData.o fusion/calvin_files/fusion/src/FusionCHPData.o fusion/calvin_files/fusion/src/FusionProbeSetResults.o fusion/calvin_files/fusion/src/GCOSAdapter/GCOSCELDataAdapter.o fusion/calvin_files/fusion/src/GCOSAdapter/GCOSCHPDataAdapter.o fusion/calvin_files/fusion/src/FusionCHPLegacyData.o fusion/calvin_files/fusion/src/FusionCHPMultiDataAccessor.o fusion/calvin_files/fusion/src/FusionCHPMultiDataData.o fusion/calvin_files/fusion/src/FusionCHPTilingData.o fusion/calvin_files/fusion/src/FusionCHPGenericData.o fusion/calvin_files/fusion/src/FusionCHPQuantificationData.o fusion/calvin_files/fusion/src/FusionCHPQuantificationDetectionData.o fusion/calvin_files/parameter/src/ParameterNameValueType.o fusion/calvin_files/parsers/src/CDFFileReader.o fusion/calvin_files/parsers/src/CelFileReader.o fusion/calvin_files/parsers/src/CHPFileReader.o fusion/calvin_files/parsers/src/CHPMultiDataFileReader.o fusion/calvin_files/parsers/src/CHPTilingFileReader.o fusion/calvin_files/parsers/src/CHPQuantificationFileReader.o fusion/calvin_files/parsers/src/CHPQuantificationDetectionFileReader.o fusion/calvin_files/parsers/src/DataGroupHeaderReader.o fusion/calvin_files/parsers/src/DataGroupReader.o fusion/calvin_files/parsers/src/DataSetHeaderReader.o fusion/calvin_files/parsers/src/DataSetReader.o fusion/calvin_files/parsers/src/FileHeaderReader.o fusion/calvin_files/parsers/src/FileInput.o fusion/calvin_files/parsers/src/GenericDataHeaderReader.o fusion/calvin_files/parsers/src/GenericFileReader.o fusion/calvin_files/utils/src/AffymetrixGuid.o fusion/calvin_files/utils/src/DateTime.o fusion/calvin_files/utils/src/FileUtils.o fusion/calvin_files/utils/src/StringUtils.o fusion/calvin_files/utils/src/checksum.o fusion/file/BPMAPFileData.o fusion/file/BPMAPFileWriter.o fusion/file/CDFFileData.o fusion/file/CELFileData.o fusion/file/CHPFileData.o fusion/file/FileIO.o fusion/file/FileWriter.o fusion/file/TsvFile/ClfFile.o fusion/file/TsvFile/PgfFile.o fusion/file/TsvFile/TsvFile.o fusion/util/AffxByteArray.o fusion/util/AffxConv.o fusion/util/MsgStream.o fusion/util/Util.o fusion/util/Err.o fusion/util/Fs.o fusion/util/Verbose.o fusion/util/RowFile.o fusion/util/TableFile.o fusion/util/Convert.o R_affx_cel_parser.o R_affx_cdf_parser.o R_affx_cdf_extras.o R_affx_bpmap_parser.o R_affx_clf_pgf_parser.o R_affx_chp_parser.o 000.init.o
g++ -std=gnu++14 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -I. -Ifusion/calvin_files/array/src -Ifusion/calvin_files/data/src -Ifusion/calvin_files/exception/src -Ifusion/calvin_files/fusion/src -Ifusion/calvin_files/fusion/src/GCOSAdapter -Ifusion/calvin_files/fusion/src/CalvinAdapter -Ifusion/calvin_files/parameter/src -Ifusion/calvin_files/parsers/src -Ifusion/calvin_files/portability/src -Ifusion/calvin_files/template/src -Ifusion/calvin_files/utils/src -Ifusion/calvin_files/writers/src -Ifusion/file -Ifusion/file/TsvFile -Ifusion/portability -Ifusion/util -Ifusion -D_USE_MEM_MAPPING_  -I/usr/local/include   -fpic  -g -O2  -Wno-sign-compare -O0 -c fusion/calvin_files/data/src/CDFData.cpp -o fusion/calvin_files/data/src/CDFData.o
g++ -std=gnu++14 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -I. -Ifusion/calvin_files/array/src -Ifusion/calvin_files/data/src -Ifusion/calvin_files/exception/src -Ifusion/calvin_files/fusion/src -Ifusion/calvin_files/fusion/src/GCOSAdapter -Ifusion/calvin_files/fusion/src/CalvinAdapter -Ifusion/calvin_files/parameter/src -Ifusion/calvin_files/parsers/src -Ifusion/calvin_files/portability/src -Ifusion/calvin_files/template/src -Ifusion/calvin_files/utils/src -Ifusion/calvin_files/writers/src -Ifusion/file -Ifusion/file/TsvFile -Ifusion/portability -Ifusion/util -Ifusion -D_USE_MEM_MAPPING_  -I/usr/local/include   -fpic  -g -O2  -Wno-sign-compare -O0 -c fusion/calvin_files/data/src/CDFProbeGroupInformation.cpp -o fusion/calvin_files/data/src/CDFProbeGroupInformation.o
g++ -std=gnu++14 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -I. -Ifusion/calvin_files/array/src -Ifusion/calvin_files/data/src -Ifusion/calvin_files/exception/src -Ifusion/calvin_files/fusion/src -Ifusion/calvin_files/fusion/src/GCOSAdapter -Ifusion/calvin_files/fusion/src/CalvinAdapter -Ifusion/calvin_files/parameter/src -Ifusion/calvin_files/parsers/src -Ifusion/calvin_files/portability/src -Ifusion/calvin_files/template/src -Ifusion/calvin_files/utils/src -Ifusion/calvin_files/writers/src -Ifusion/file -Ifusion/file/TsvFile -Ifusion/portability -Ifusion/util -Ifusion -D_USE_MEM_MAPPING_  -I/usr/local/include   -fpic  -g -O2  -Wno-sign-compare -O0 -c fusion/calvin_files/data/src/CDFProbeInformation.cpp -o fusion/calvin_files/data/src/CDFProbeInformation.o
g++ -std=gnu++14 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -I. -Ifusion/calvin_files/array/src -Ifusion/calvin_files/data/src -Ifusion/calvin_files/exception/src -Ifusion/calvin_files/fusion/src -Ifusion/calvin_files/fusion/src/GCOSAdapter -Ifusion/calvin_files/fusion/src/CalvinAdapter -Ifusion/calvin_files/parameter/src -Ifusion/calvin_files/parsers/src -Ifusion/calvin_files/portability/src -Ifusion/calvin_files/template/src -Ifusion/calvin_files/utils/src -Ifusion/calvin_files/writers/src -Ifusion/file -Ifusion/file/TsvFile -Ifusion/portability -Ifusion/util -Ifusion -D_USE_MEM_MAPPING_  -I/usr/local/include   -fpic  -g -O2  -Wno-sign-compare -O0 -c fusion/calvin_files/data/src/CDFProbeSetInformation.cpp -o fusion/calvin_files/data/src/CDFProbeSetInformation.o
g++ -std=gnu++14 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -I. -Ifusion/calvin_files/array/src -Ifusion/calvin_files/data/src -Ifusion/calvin_files/exception/src -Ifusion/calvin_files/fusion/src -Ifusion/calvin_files/fusion/src/GCOSAdapter -Ifusion/calvin_files/fusion/src/CalvinAdapter -Ifusion/calvin_files/parameter/src -Ifusion/calvin_files/parsers/src -Ifusion/calvin_files/portability/src -Ifusion/calvin_files/template/src -Ifusion/calvin_files/utils/src -Ifusion/calvin_files/writers/src -Ifusion/file -Ifusion/file/TsvFile -Ifusion/portability -Ifusion/util -Ifusion -D_USE_MEM_MAPPING_  -I/usr/local/include   -fpic  -g -O2  -Wno-sign-compare -O0 -c fusion/calvin_files/data/src/CDFQCProbeInformation.cpp -o fusion/calvin_files/data/src/CDFQCProbeInformation.o
g++ -std=gnu++14 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -I. -Ifusion/calvin_files/array/src -Ifusion/calvin_files/data/src -Ifusion/calvin_files/exception/src -Ifusion/calvin_files/fusion/src -Ifusion/calvin_files/fusion/src/GCOSAdapter -Ifusion/calvin_files/fusion/src/CalvinAdapter -Ifusion/calvin_files/parameter/src -Ifusion/calvin_files/parsers/src -Ifusion/calvin_files/portability/src -Ifusion/calvin_files/template/src -Ifusion/calvin_files/utils/src -Ifusion/calvin_files/writers/src -Ifusion/file -Ifusion/file/TsvFile -Ifusion/portability -Ifusion/util -Ifusion -D_USE_MEM_MAPPING_  -I/usr/local/include   -fpic  -g -O2  -Wno-sign-compare -O0 -c fusion/calvin_files/data/src/CDFQCProbeSetInformation.cpp -o fusion/calvin_files/data/src/CDFQCProbeSetInformation.o
g++ -std=gnu++14 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -I. -Ifusion/calvin_files/array/src -Ifusion/calvin_files/data/src -Ifusion/calvin_files/exception/src -Ifusion/calvin_files/fusion/src -Ifusion/calvin_files/fusion/src/GCOSAdapter -Ifusion/calvin_files/fusion/src/CalvinAdapter -Ifusion/calvin_files/parameter/src -Ifusion/calvin_files/parsers/src -Ifusion/calvin_files/portability/src -Ifusion/calvin_files/template/src -Ifusion/calvin_files/utils/src -Ifusion/calvin_files/writers/src -Ifusion/file -Ifusion/file/TsvFile -Ifusion/portability -Ifusion/util -Ifusion -D_USE_MEM_MAPPING_  -I/usr/local/include   -fpic  -g -O2  -Wno-sign-compare -O0 -c fusion/calvin_files/data/src/CELData.cpp -o fusion/calvin_files/data/src/CELData.o
g++ -std=gnu++14 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -I. -Ifusion/calvin_files/array/src -Ifusion/calvin_files/data/src -Ifusion/calvin_files/exception/src -Ifusion/calvin_files/fusion/src -Ifusion/calvin_files/fusion/src/GCOSAdapter -Ifusion/calvin_files/fusion/src/CalvinAdapter -Ifusion/calvin_files/parameter/src -Ifusion/calvin_files/parsers/src -Ifusion/calvin_files/portability/src -Ifusion/calvin_files/template/src -Ifusion/calvin_files/utils/src -Ifusion/calvin_files/writers/src -Ifusion/file -Ifusion/file/TsvFile -Ifusion/portability -Ifusion/util -Ifusion -D_USE_MEM_MAPPING_  -I/usr/local/include   -fpic  -g -O2  -Wno-sign-compare -O0 -c fusion/calvin_files/data/src/CHPBackgroundZone.cpp -o fusion/calvin_files/data/src/CHPBackgroundZone.o
g++ -std=gnu++14 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -I. -Ifusion/calvin_files/array/src -Ifusion/calvin_files/data/src -Ifusion/calvin_files/exception/src -Ifusion/calvin_files/fusion/src -Ifusion/calvin_files/fusion/src/GCOSAdapter -Ifusion/calvin_files/fusion/src/CalvinAdapter -Ifusion/calvin_files/parameter/src -Ifusion/calvin_files/parsers/src -Ifusion/calvin_files/portability/src -Ifusion/calvin_files/template/src -Ifusion/calvin_files/utils/src -Ifusion/calvin_files/writers/src -Ifusion/file -Ifusion/file/TsvFile -Ifusion/portability -Ifusion/util -Ifusion -D_USE_MEM_MAPPING_  -I/usr/local/include   -fpic  -g -O2  -Wno-sign-compare -O0 -c fusion/calvin_files/data/src/CHPData.cpp -o fusion/calvin_files/data/src/CHPData.o
...
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -D_FILE_OFFSET_BITS=64 -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/S4Vectors/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/IRanges/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/XVector/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/Biostrings/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/Rhtslib/include' -I/usr/local/include   -fpic  -g -O2  -c strhash.c -o strhash.o
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -D_FILE_OFFSET_BITS=64 -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/S4Vectors/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/IRanges/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/XVector/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/Biostrings/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/Rhtslib/include' -I/usr/local/include   -fpic  -g -O2  -c utilities.c -o utilities.o
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -D_FILE_OFFSET_BITS=64 -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/S4Vectors/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/IRanges/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/XVector/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/Biostrings/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/Rhtslib/include' -I/usr/local/include   -fpic  -g -O2  -c vcffile.c -o vcffile.o
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -D_FILE_OFFSET_BITS=64 -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/S4Vectors/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/IRanges/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/XVector/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/Biostrings/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/Rhtslib/include' -I/usr/local/include   -fpic  -g -O2  -c vcftype.c -o vcftype.o
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -D_FILE_OFFSET_BITS=64 -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/S4Vectors/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/IRanges/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/XVector/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/Biostrings/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/CINdex/Rhtslib/include' -I/usr/local/include   -fpic  -g -O2  -c writevcf.c -o writevcf.o
gcc -shared -L/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/lib -L/usr/local/lib64 -o VariantAnnotation.so Biostrings_stubs.o IRanges_stubs.o R_init_VariantAnnotation.o XVector_stubs.o dna_hash.o rle.o strhash.o utilities.o vcffile.o vcftype.o writevcf.o /c4/home/henrik/repositories/R.utils/revdep/library/CINdex/Rhtslib/usrlib/libhts.a -lcurl -L/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/lib -lR
Creating a new generic function for ‘tabulate’ in package ‘VariantAnnotation’
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG   -I/usr/local/include   -fpic  -g -O2  -c R_init_biovizBase.c -o R_init_biovizBase.o
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG   -I/usr/local/include   -fpic  -g -O2  -c bin_offsets.c -o bin_offsets.o
gcc -shared -L/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/lib -L/usr/local/lib64 -o biovizBase.so R_init_biovizBase.o bin_offsets.o -L/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/lib -lR


trying URL 'https://bioconductor.org/packages/3.15/bioc/src/contrib/enrichplot_1.16.1.tar.gz'
Content type 'application/x-gzip' length 93082 bytes (90 KB)
==================================================
downloaded 90 KB

trying URL 'https://bioconductor.org/packages/3.15/bioc/src/contrib/graphite_1.42.0.tar.gz'
Content type 'application/x-gzip' length 780805 bytes (762 KB)
==================================================
downloaded 762 KB

...
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (ReactomePA)

The downloaded source packages are in
	‘/scratch/henrik/641569/RtmphNjpta/downloaded_packages’


```
# ELMER

<details>

* Version: 2.20.0
* GitHub: NA
* Source code: https://github.com/cran/ELMER
* Date/Publication: 2022-04-26
* Number of recursive dependencies: 228

Run `revdep_details(, "ELMER")` for more info

</details>

## In both

*   checking whether package ‘ELMER’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/c4/home/henrik/repositories/R.utils/revdep/checks/ELMER/new/ELMER.Rcheck/00install.out’ for details.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘GenomicInteractions’
    ```

## Installation

### Devel

```
* installing *source* package ‘ELMER’ ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘RSQLite’
Calls: <Anonymous> ... loadNamespace -> withRestarts -> withOneRestart -> doWithOneRestart
Execution halted
ERROR: lazy loading failed for package ‘ELMER’
* removing ‘/c4/home/henrik/repositories/R.utils/revdep/checks/ELMER/new/ELMER.Rcheck/ELMER’


```
### CRAN

```
* installing *source* package ‘ELMER’ ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘RSQLite’
Calls: <Anonymous> ... loadNamespace -> withRestarts -> withOneRestart -> doWithOneRestart
Execution halted
ERROR: lazy loading failed for package ‘ELMER’
* removing ‘/c4/home/henrik/repositories/R.utils/revdep/checks/ELMER/old/ELMER.Rcheck/ELMER’


```
# enrichTF

<details>

* Version: 1.12.0
* GitHub: https://github.com/wzthu/enrichTF
* Source code: https://github.com/cran/enrichTF
* Date/Publication: 2022-04-26
* Number of recursive dependencies: 211

Run `revdep_details(, "enrichTF")` for more info

</details>

## In both

*   checking whether package ‘enrichTF’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/c4/home/henrik/repositories/R.utils/revdep/checks/enrichTF/new/enrichTF.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘enrichTF’ ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
Error in loadNamespace(j <- i[[1L]], c(lib.loc, .libPaths()), versionCheck = vI[[j]]) : 
  there is no package called ‘RSQLite’
Calls: <Anonymous> ... loadNamespace -> withRestarts -> withOneRestart -> doWithOneRestart
Execution halted
ERROR: lazy loading failed for package ‘enrichTF’
* removing ‘/c4/home/henrik/repositories/R.utils/revdep/checks/enrichTF/new/enrichTF.Rcheck/enrichTF’


```
### CRAN

```
* installing *source* package ‘enrichTF’ ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
Error in loadNamespace(j <- i[[1L]], c(lib.loc, .libPaths()), versionCheck = vI[[j]]) : 
  there is no package called ‘RSQLite’
Calls: <Anonymous> ... loadNamespace -> withRestarts -> withOneRestart -> doWithOneRestart
Execution halted
ERROR: lazy loading failed for package ‘enrichTF’
* removing ‘/c4/home/henrik/repositories/R.utils/revdep/checks/enrichTF/old/enrichTF.Rcheck/enrichTF’


```
# esATAC

<details>

* Version: 1.18.0
* GitHub: https://github.com/wzthu/esATAC
* Source code: https://github.com/cran/esATAC
* Date/Publication: 2022-04-26
* Number of recursive dependencies: 207

Run `revdep_details(, "esATAC")` for more info

</details>

## In both

*   checking whether package ‘esATAC’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/c4/home/henrik/repositories/R.utils/revdep/checks/esATAC/new/esATAC.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘esATAC’ ...
** using staged installation
** libs
g++ -std=gnu++11 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -DNDEBUG -DPLF_SYS_LINUX  -DR_EVN_FLAG -I'/c4/home/henrik/repositories/R.utils/revdep/library/esATAC/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -c BedLine.cpp -o BedLine.o
g++ -std=gnu++11 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -DNDEBUG -DPLF_SYS_LINUX  -DR_EVN_FLAG -I'/c4/home/henrik/repositories/R.utils/revdep/library/esATAC/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -c BedUtils.cpp -o BedUtils.o
g++ -std=gnu++11 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -DNDEBUG -DPLF_SYS_LINUX  -DR_EVN_FLAG -I'/c4/home/henrik/repositories/R.utils/revdep/library/esATAC/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -c ChrDivi.cpp -o ChrDivi.o
g++ -std=gnu++11 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -DNDEBUG -DPLF_SYS_LINUX  -DR_EVN_FLAG -I'/c4/home/henrik/repositories/R.utils/revdep/library/esATAC/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -c CutCountPre.cpp -o CutCountPre.o
g++ -std=gnu++11 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -DNDEBUG -DPLF_SYS_LINUX  -DR_EVN_FLAG -I'/c4/home/henrik/repositories/R.utils/revdep/library/esATAC/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -c CutSiteCount.cpp -o CutSiteCount.o
g++ -std=gnu++11 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -DNDEBUG -DPLF_SYS_LINUX  -DR_EVN_FLAG -I'/c4/home/henrik/repositories/R.utils/revdep/library/esATAC/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -c LibComplexQC.cpp -o LibComplexQC.o
g++ -std=gnu++11 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -DNDEBUG -DPLF_SYS_LINUX  -DR_EVN_FLAG -I'/c4/home/henrik/repositories/R.utils/revdep/library/esATAC/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -c RcppExports.cpp -o RcppExports.o
...
installing to /c4/home/henrik/repositories/R.utils/revdep/checks/esATAC/new/esATAC.Rcheck/00LOCK-esATAC/00new/esATAC/libs
** R
** inst
** byte-compile and prepare package for lazy loading
Error in loadNamespace(j <- i[[1L]], c(lib.loc, .libPaths()), versionCheck = vI[[j]]) : 
  there is no package called ‘tibble’
Calls: <Anonymous> ... loadNamespace -> withRestarts -> withOneRestart -> doWithOneRestart
Execution halted
ERROR: lazy loading failed for package ‘esATAC’
* removing ‘/c4/home/henrik/repositories/R.utils/revdep/checks/esATAC/new/esATAC.Rcheck/esATAC’


```
### CRAN

```
* installing *source* package ‘esATAC’ ...
** using staged installation
** libs
g++ -std=gnu++11 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -DNDEBUG -DPLF_SYS_LINUX  -DR_EVN_FLAG -I'/c4/home/henrik/repositories/R.utils/revdep/library/esATAC/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -c BedLine.cpp -o BedLine.o
g++ -std=gnu++11 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -DNDEBUG -DPLF_SYS_LINUX  -DR_EVN_FLAG -I'/c4/home/henrik/repositories/R.utils/revdep/library/esATAC/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -c BedUtils.cpp -o BedUtils.o
g++ -std=gnu++11 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -DNDEBUG -DPLF_SYS_LINUX  -DR_EVN_FLAG -I'/c4/home/henrik/repositories/R.utils/revdep/library/esATAC/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -c ChrDivi.cpp -o ChrDivi.o
g++ -std=gnu++11 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -DNDEBUG -DPLF_SYS_LINUX  -DR_EVN_FLAG -I'/c4/home/henrik/repositories/R.utils/revdep/library/esATAC/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -c CutCountPre.cpp -o CutCountPre.o
g++ -std=gnu++11 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -DNDEBUG -DPLF_SYS_LINUX  -DR_EVN_FLAG -I'/c4/home/henrik/repositories/R.utils/revdep/library/esATAC/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -c CutSiteCount.cpp -o CutSiteCount.o
g++ -std=gnu++11 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -DNDEBUG -DPLF_SYS_LINUX  -DR_EVN_FLAG -I'/c4/home/henrik/repositories/R.utils/revdep/library/esATAC/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -c LibComplexQC.cpp -o LibComplexQC.o
g++ -std=gnu++11 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG -DNDEBUG -DPLF_SYS_LINUX  -DR_EVN_FLAG -I'/c4/home/henrik/repositories/R.utils/revdep/library/esATAC/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -c RcppExports.cpp -o RcppExports.o
...
installing to /c4/home/henrik/repositories/R.utils/revdep/checks/esATAC/old/esATAC.Rcheck/00LOCK-esATAC/00new/esATAC/libs
** R
** inst
** byte-compile and prepare package for lazy loading
Error in loadNamespace(j <- i[[1L]], c(lib.loc, .libPaths()), versionCheck = vI[[j]]) : 
  there is no package called ‘tibble’
Calls: <Anonymous> ... loadNamespace -> withRestarts -> withOneRestart -> doWithOneRestart
Execution halted
ERROR: lazy loading failed for package ‘esATAC’
* removing ‘/c4/home/henrik/repositories/R.utils/revdep/checks/esATAC/old/esATAC.Rcheck/esATAC’


```
# fastreeR

<details>

* Version: 1.0.0
* GitHub: https://github.com/gkanogiannis/fastreeR
* Source code: https://github.com/cran/fastreeR
* Date/Publication: 2022-04-26
* Number of recursive dependencies: 87

Run `revdep_details(, "fastreeR")` for more info

</details>

## In both

*   R CMD check timed out
    

# htmldf

<details>

* Version: 0.5.0
* GitHub: https://github.com/alastairrushworth/htmldf
* Source code: https://github.com/cran/htmldf
* Date/Publication: 2022-01-13 11:12:43 UTC
* Number of recursive dependencies: 63

Run `revdep_details(, "htmldf")` for more info

</details>

## In both

*   checking whether package ‘htmldf’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/c4/home/henrik/repositories/R.utils/revdep/checks/htmldf/new/htmldf.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘htmldf’ ...
** package ‘htmldf’ successfully unpacked and MD5 sums checked
** using staged installation
** R
** byte-compile and prepare package for lazy loading
Error in loadNamespace(j <- i[[1L]], c(lib.loc, .libPaths()), versionCheck = vI[[j]]) : 
  there is no package called ‘pillar’
Calls: <Anonymous> ... loadNamespace -> withRestarts -> withOneRestart -> doWithOneRestart
Execution halted
ERROR: lazy loading failed for package ‘htmldf’
* removing ‘/c4/home/henrik/repositories/R.utils/revdep/checks/htmldf/new/htmldf.Rcheck/htmldf’


```
### CRAN

```
* installing *source* package ‘htmldf’ ...
** package ‘htmldf’ successfully unpacked and MD5 sums checked
** using staged installation
** R
** byte-compile and prepare package for lazy loading
Error in loadNamespace(j <- i[[1L]], c(lib.loc, .libPaths()), versionCheck = vI[[j]]) : 
  there is no package called ‘pillar’
Calls: <Anonymous> ... loadNamespace -> withRestarts -> withOneRestart -> doWithOneRestart
Execution halted
ERROR: lazy loading failed for package ‘htmldf’
* removing ‘/c4/home/henrik/repositories/R.utils/revdep/checks/htmldf/old/htmldf.Rcheck/htmldf’


```
# HyPhy

<details>

* Version: 1.0
* GitHub: NA
* Source code: https://github.com/cran/HyPhy
* Date/Publication: 2012-07-30 04:09:01
* Number of recursive dependencies: 7

Run `revdep_details(, "HyPhy")` for more info

</details>

## In both

*   checking whether package ‘HyPhy’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/c4/home/henrik/repositories/R.utils/revdep/checks/HyPhy/new/HyPhy.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘HyPhy’ ...
** package ‘HyPhy’ successfully unpacked and MD5 sums checked
** using staged installation
** R
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘ape’ in loadNamespace(j <- i[[1L]], c(lib.loc, .libPaths()), versionCheck = vI[[j]]):
 there is no package called ‘Rcpp’
Execution halted
ERROR: lazy loading failed for package ‘HyPhy’
* removing ‘/c4/home/henrik/repositories/R.utils/revdep/checks/HyPhy/new/HyPhy.Rcheck/HyPhy’


```
### CRAN

```
* installing *source* package ‘HyPhy’ ...
** package ‘HyPhy’ successfully unpacked and MD5 sums checked
** using staged installation
** R
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘ape’ in loadNamespace(j <- i[[1L]], c(lib.loc, .libPaths()), versionCheck = vI[[j]]):
 there is no package called ‘Rcpp’
Execution halted
ERROR: lazy loading failed for package ‘HyPhy’
* removing ‘/c4/home/henrik/repositories/R.utils/revdep/checks/HyPhy/old/HyPhy.Rcheck/HyPhy’


```
# muscData

<details>

* Version: 
* GitHub: https://github.com/HenrikBengtsson/R.utils
* Source code: NA
* Number of recursive dependencies: 0

</details>

## Error before installation

### Devel

```
Creating a new generic function for ‘append’ in package ‘BiocGenerics’
Creating a new generic function for ‘as.data.frame’ in package ‘BiocGenerics’
Creating a new generic function for ‘cbind’ in package ‘BiocGenerics’
Creating a new generic function for ‘rbind’ in package ‘BiocGenerics’
Creating a new generic function for ‘do.call’ in package ‘BiocGenerics’
Creating a new generic function for ‘duplicated’ in package ‘BiocGenerics’
Creating a new generic function for ‘anyDuplicated’ in package ‘BiocGenerics’
Creating a new generic function for ‘eval’ in package ‘BiocGenerics’
Creating a new generic function for ‘pmax’ in package ‘BiocGenerics’
Creating a new generic function for ‘pmin’ in package ‘BiocGenerics’
...
Warning in fun(libname, pkgname) :
  rgeos: versions of GEOS runtime 3.9.1-CAPI-1.14.2
and GEOS at installation 3.4.2-CAPI-1.8.2differ
Warning in fun(libname, pkgname) :
  rgeos: versions of GEOS runtime 3.9.1-CAPI-1.14.2
and GEOS at installation 3.4.2-CAPI-1.8.2differ
g++ -std=gnu++14 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/muscData/Rcpp/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/muscData/RcppArmadillo/include' -I/usr/local/include   -fpic  -g -O2  -c DESeq2.cpp -o DESeq2.o
g++ -std=gnu++14 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/muscData/Rcpp/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/muscData/RcppArmadillo/include' -I/usr/local/include   -fpic  -g -O2  -c RcppExports.cpp -o RcppExports.o
g++ -std=gnu++14 -shared -L/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/lib -L/usr/local/lib64 -o DESeq2.so DESeq2.o RcppExports.o -L/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/lib -lRlapack -L/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/lib -lRblas -lgfortran -lm -lquadmath -L/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/lib -lR
g++ -std=gnu++14 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/muscData/Rcpp/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/muscData/RcppEigen/include' -I/usr/local/include  -DNDEBUG -DEIGEN_DONT_VECTORIZE -fpic  -g -O2  -c external.cpp -o external.o


* installing *binary* package ‘abind’ ...
* DONE (abind)
* installing *binary* package ‘aod’ ...
* DONE (aod)
* installing *binary* package ‘assertthat’ ...
* DONE (assertthat)
* installing *binary* package ‘backports’ ...
* DONE (backports)
* installing *binary* package ‘base64enc’ ...
* DONE (base64enc)
...
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (ExperimentHub)
* installing *source* package ‘lme4’ ...
** package ‘lme4’ successfully unpacked and MD5 sums checked
** using staged installation
** libs


```
### CRAN

```
Creating a new generic function for ‘append’ in package ‘BiocGenerics’
Creating a new generic function for ‘as.data.frame’ in package ‘BiocGenerics’
Creating a new generic function for ‘cbind’ in package ‘BiocGenerics’
Creating a new generic function for ‘rbind’ in package ‘BiocGenerics’
Creating a new generic function for ‘do.call’ in package ‘BiocGenerics’
Creating a new generic function for ‘duplicated’ in package ‘BiocGenerics’
Creating a new generic function for ‘anyDuplicated’ in package ‘BiocGenerics’
Creating a new generic function for ‘eval’ in package ‘BiocGenerics’
Creating a new generic function for ‘pmax’ in package ‘BiocGenerics’
Creating a new generic function for ‘pmin’ in package ‘BiocGenerics’
...
Warning in fun(libname, pkgname) :
  rgeos: versions of GEOS runtime 3.9.1-CAPI-1.14.2
and GEOS at installation 3.4.2-CAPI-1.8.2differ
Warning in fun(libname, pkgname) :
  rgeos: versions of GEOS runtime 3.9.1-CAPI-1.14.2
and GEOS at installation 3.4.2-CAPI-1.8.2differ
g++ -std=gnu++14 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/muscData/Rcpp/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/muscData/RcppArmadillo/include' -I/usr/local/include   -fpic  -g -O2  -c DESeq2.cpp -o DESeq2.o
g++ -std=gnu++14 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/muscData/Rcpp/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/muscData/RcppArmadillo/include' -I/usr/local/include   -fpic  -g -O2  -c RcppExports.cpp -o RcppExports.o
g++ -std=gnu++14 -shared -L/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/lib -L/usr/local/lib64 -o DESeq2.so DESeq2.o RcppExports.o -L/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/lib -lRlapack -L/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/lib -lRblas -lgfortran -lm -lquadmath -L/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/lib -lR
g++ -std=gnu++14 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/muscData/Rcpp/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/muscData/RcppEigen/include' -I/usr/local/include  -DNDEBUG -DEIGEN_DONT_VECTORIZE -fpic  -g -O2  -c external.cpp -o external.o


* installing *binary* package ‘abind’ ...
* DONE (abind)
* installing *binary* package ‘aod’ ...
* DONE (aod)
* installing *binary* package ‘assertthat’ ...
* DONE (assertthat)
* installing *binary* package ‘backports’ ...
* DONE (backports)
* installing *binary* package ‘base64enc’ ...
* DONE (base64enc)
...
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (ExperimentHub)
* installing *source* package ‘lme4’ ...
** package ‘lme4’ successfully unpacked and MD5 sums checked
** using staged installation
** libs


```
# OSCA.intro

<details>

* Version: 
* GitHub: https://github.com/HenrikBengtsson/R.utils
* Source code: NA
* Number of recursive dependencies: 0

</details>

## Error before installation

### Devel

```
Creating a new generic function for ‘append’ in package ‘BiocGenerics’
Creating a new generic function for ‘as.data.frame’ in package ‘BiocGenerics’
Creating a new generic function for ‘cbind’ in package ‘BiocGenerics’
Creating a new generic function for ‘rbind’ in package ‘BiocGenerics’
Creating a new generic function for ‘do.call’ in package ‘BiocGenerics’
Creating a new generic function for ‘duplicated’ in package ‘BiocGenerics’
Creating a new generic function for ‘anyDuplicated’ in package ‘BiocGenerics’
Creating a new generic function for ‘eval’ in package ‘BiocGenerics’
Creating a new generic function for ‘pmax’ in package ‘BiocGenerics’
Creating a new generic function for ‘pmin’ in package ‘BiocGenerics’
...
g++ -std=gnu++11 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/Rcpp/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/beachmat/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/BH/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/dqrng/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/scuttle/include' -I/usr/local/include   -fpic  -g -O2  -c RcppExports.cpp -o RcppExports.o
g++ -std=gnu++11 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/Rcpp/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/beachmat/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/BH/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/dqrng/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/scuttle/include' -I/usr/local/include   -fpic  -g -O2  -c choose_effect_size.cpp -o choose_effect_size.o
g++ -std=gnu++11 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/Rcpp/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/beachmat/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/BH/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/dqrng/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/scuttle/include' -I/usr/local/include   -fpic  -g -O2  -c combine_rho.cpp -o combine_rho.o
g++ -std=gnu++11 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/Rcpp/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/beachmat/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/BH/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/dqrng/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/scuttle/include' -I/usr/local/include   -fpic  -g -O2  -c compute_blocked_stats.cpp -o compute_blocked_stats.o
g++ -std=gnu++11 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/Rcpp/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/beachmat/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/BH/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/dqrng/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/scuttle/include' -I/usr/local/include   -fpic  -g -O2  -c compute_residual_stats.cpp -o compute_residual_stats.o
g++ -std=gnu++11 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/Rcpp/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/beachmat/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/BH/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/dqrng/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/scuttle/include' -I/usr/local/include   -fpic  -g -O2  -c compute_rho_null.cpp -o compute_rho_null.o
g++ -std=gnu++11 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/Rcpp/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/beachmat/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/BH/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/dqrng/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/scuttle/include' -I/usr/local/include   -fpic  -g -O2  -c cyclone_scores.cpp -o cyclone_scores.o
g++ -std=gnu++11 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/Rcpp/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/beachmat/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/BH/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/dqrng/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/scuttle/include' -I/usr/local/include   -fpic  -g -O2  -c overlap_exprs.cpp -o overlap_exprs.o
g++ -std=gnu++11 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/Rcpp/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/beachmat/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/BH/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/dqrng/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/scuttle/include' -I/usr/local/include   -fpic  -g -O2  -c rand_custom.cpp -o rand_custom.o
g++ -std=gnu++11 -shared -L/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/lib -L/usr/local/lib64 -o scran.so RcppExports.o choose_effect_size.o combine_rho.o compute_blocked_stats.o compute_residual_stats.o compute_rho_null.o cyclone_scores.o overlap_exprs.o rand_custom.o -L/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/lib -lRlapack -L/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/lib -lRblas -lgfortran -lm -lquadmath -L/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/lib -lR


trying URL 'https://bioconductor.org/packages/3.15/data/experiment/src/contrib/scRNAseq_2.10.0.tar.gz'
Content type 'application/x-gzip' length 349138 bytes (340 KB)
==================================================
downloaded 340 KB

trying URL 'https://bioconductor.org/packages/3.15/bioc/src/contrib/zellkonverter_1.6.1.tar.gz'
Content type 'application/x-gzip' length 1131745 bytes (1.1 MB)
==================================================
downloaded 1.1 MB

...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location


```
### CRAN

```
Creating a new generic function for ‘append’ in package ‘BiocGenerics’
Creating a new generic function for ‘as.data.frame’ in package ‘BiocGenerics’
Creating a new generic function for ‘cbind’ in package ‘BiocGenerics’
Creating a new generic function for ‘rbind’ in package ‘BiocGenerics’
Creating a new generic function for ‘do.call’ in package ‘BiocGenerics’
Creating a new generic function for ‘duplicated’ in package ‘BiocGenerics’
Creating a new generic function for ‘anyDuplicated’ in package ‘BiocGenerics’
Creating a new generic function for ‘eval’ in package ‘BiocGenerics’
Creating a new generic function for ‘pmax’ in package ‘BiocGenerics’
Creating a new generic function for ‘pmin’ in package ‘BiocGenerics’
...
g++ -std=gnu++11 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/Rcpp/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/beachmat/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/BH/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/dqrng/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/scuttle/include' -I/usr/local/include   -fpic  -g -O2  -c RcppExports.cpp -o RcppExports.o
g++ -std=gnu++11 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/Rcpp/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/beachmat/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/BH/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/dqrng/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/scuttle/include' -I/usr/local/include   -fpic  -g -O2  -c choose_effect_size.cpp -o choose_effect_size.o
g++ -std=gnu++11 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/Rcpp/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/beachmat/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/BH/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/dqrng/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/scuttle/include' -I/usr/local/include   -fpic  -g -O2  -c combine_rho.cpp -o combine_rho.o
g++ -std=gnu++11 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/Rcpp/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/beachmat/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/BH/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/dqrng/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/scuttle/include' -I/usr/local/include   -fpic  -g -O2  -c compute_blocked_stats.cpp -o compute_blocked_stats.o
g++ -std=gnu++11 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/Rcpp/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/beachmat/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/BH/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/dqrng/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/scuttle/include' -I/usr/local/include   -fpic  -g -O2  -c compute_residual_stats.cpp -o compute_residual_stats.o
g++ -std=gnu++11 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/Rcpp/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/beachmat/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/BH/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/dqrng/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/scuttle/include' -I/usr/local/include   -fpic  -g -O2  -c compute_rho_null.cpp -o compute_rho_null.o
g++ -std=gnu++11 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/Rcpp/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/beachmat/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/BH/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/dqrng/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/scuttle/include' -I/usr/local/include   -fpic  -g -O2  -c cyclone_scores.cpp -o cyclone_scores.o
g++ -std=gnu++11 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/Rcpp/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/beachmat/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/BH/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/dqrng/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/scuttle/include' -I/usr/local/include   -fpic  -g -O2  -c overlap_exprs.cpp -o overlap_exprs.o
g++ -std=gnu++11 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/Rcpp/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/beachmat/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/BH/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/dqrng/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/OSCA.intro/scuttle/include' -I/usr/local/include   -fpic  -g -O2  -c rand_custom.cpp -o rand_custom.o
g++ -std=gnu++11 -shared -L/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/lib -L/usr/local/lib64 -o scran.so RcppExports.o choose_effect_size.o combine_rho.o compute_blocked_stats.o compute_residual_stats.o compute_rho_null.o cyclone_scores.o overlap_exprs.o rand_custom.o -L/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/lib -lRlapack -L/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/lib -lRblas -lgfortran -lm -lquadmath -L/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/lib -lR


trying URL 'https://bioconductor.org/packages/3.15/data/experiment/src/contrib/scRNAseq_2.10.0.tar.gz'
Content type 'application/x-gzip' length 349138 bytes (340 KB)
==================================================
downloaded 340 KB

trying URL 'https://bioconductor.org/packages/3.15/bioc/src/contrib/zellkonverter_1.6.1.tar.gz'
Content type 'application/x-gzip' length 1131745 bytes (1.1 MB)
==================================================
downloaded 1.1 MB

...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location


```
# R.devices

<details>

* Version: 2.17.0
* GitHub: https://github.com/HenrikBengtsson/R.devices
* Source code: https://github.com/cran/R.devices
* Date/Publication: 2021-01-19 18:30:06 UTC
* Number of recursive dependencies: 8

Run `revdep_details(, "R.devices")` for more info

</details>

## Newly broken

*   checking whether package ‘R.devices’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/c4/home/henrik/repositories/R.utils/revdep/checks/R.devices/new/R.devices.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘R.devices’ ...
** package ‘R.devices’ successfully unpacked and MD5 sums checked
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
Error: object ‘inherits’ is not exported by 'namespace:R.utils'
Execution halted
ERROR: lazy loading failed for package ‘R.devices’
* removing ‘/c4/home/henrik/repositories/R.utils/revdep/checks/R.devices/new/R.devices.Rcheck/R.devices’


```
### CRAN

```
* installing *source* package ‘R.devices’ ...
** package ‘R.devices’ successfully unpacked and MD5 sums checked
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (R.devices)


```
# R.rsp

<details>

* Version: 0.44.0
* GitHub: https://github.com/HenrikBengtsson/R.rsp
* Source code: https://github.com/cran/R.rsp
* Date/Publication: 2020-07-09 16:20:02 UTC
* Number of recursive dependencies: 18

Run `revdep_details(, "R.rsp")` for more info

</details>

## Newly broken

*   checking whether package ‘R.rsp’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/c4/home/henrik/repositories/R.utils/revdep/checks/R.rsp/new/R.rsp.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘R.rsp’ ...
** package ‘R.rsp’ successfully unpacked and MD5 sums checked
** using staged installation
** R
** exec
** inst
** byte-compile and prepare package for lazy loading
Error: object ‘inherits’ is not exported by 'namespace:R.utils'
Execution halted
ERROR: lazy loading failed for package ‘R.rsp’
* removing ‘/c4/home/henrik/repositories/R.utils/revdep/checks/R.rsp/new/R.rsp.Rcheck/R.rsp’


```
### CRAN

```
* installing *source* package ‘R.rsp’ ...
** package ‘R.rsp’ successfully unpacked and MD5 sums checked
** using staged installation
** R
** exec
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (R.rsp)


```
# RforProteomics

<details>

* Version: 
* GitHub: https://github.com/HenrikBengtsson/R.utils
* Source code: NA
* Number of recursive dependencies: 0

</details>

## Error before installation

### Devel

```
Creating a new generic function for ‘append’ in package ‘BiocGenerics’
Creating a new generic function for ‘as.data.frame’ in package ‘BiocGenerics’
Creating a new generic function for ‘cbind’ in package ‘BiocGenerics’
Creating a new generic function for ‘rbind’ in package ‘BiocGenerics’
Creating a new generic function for ‘do.call’ in package ‘BiocGenerics’
Creating a new generic function for ‘duplicated’ in package ‘BiocGenerics’
Creating a new generic function for ‘anyDuplicated’ in package ‘BiocGenerics’
Creating a new generic function for ‘eval’ in package ‘BiocGenerics’
Creating a new generic function for ‘pmax’ in package ‘BiocGenerics’
Creating a new generic function for ‘pmin’ in package ‘BiocGenerics’
...
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/RforProteomics/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -c Spectrum1_class.c -o Spectrum1_class.o
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/RforProteomics/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -c Spectrum2_class.c -o Spectrum2_class.o
g++ -std=gnu++14 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/RforProteomics/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -c imp_neighbour_avg.cpp -o imp_neighbour_avg.o
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/RforProteomics/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -c utils.c -o utils.o
g++ -std=gnu++14 -shared -L/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/lib -L/usr/local/lib64 -o MSnbase.so RcppExports.o Spectrum1_class.o Spectrum2_class.o imp_neighbour_avg.o utils.o -L/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/lib -lR
Creating a new generic function for ‘trimws’ in package ‘MSnbase’
in method for ‘coerce’ with signature ‘"MSnSet","SummarizedExperiment"’: no definition for class “SummarizedExperiment”
in method for ‘coerce’ with signature ‘"SummarizedExperiment","MSnSet"’: no definition for class “SummarizedExperiment”
in method for ‘coerce’ with signature ‘"IBSpectra","MSnSet"’: no definition for class “IBSpectra”
in method for ‘plotMzDelta’ with signature ‘"mzRramp"’: no definition for class “mzRramp”


trying URL 'https://bioconductor.org/packages/3.15/bioc/src/contrib/MLInterfaces_1.76.0.tar.gz'
Content type 'application/x-gzip' length 1374688 bytes (1.3 MB)
==================================================
downloaded 1.3 MB

trying URL 'https://bioconductor.org/packages/3.15/data/experiment/src/contrib/msdata_0.36.0.tar.gz'
Content type 'application/x-gzip' length 447124112 bytes (426.4 MB)
==================================================
downloaded 426.4 MB

...
* DONE (biomaRt)
* installing *binary* package ‘caret’ ...
* DONE (caret)
* installing *source* package ‘DEP’ ...
** using staged installation
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading


```
### CRAN

```
Creating a new generic function for ‘append’ in package ‘BiocGenerics’
Creating a new generic function for ‘as.data.frame’ in package ‘BiocGenerics’
Creating a new generic function for ‘cbind’ in package ‘BiocGenerics’
Creating a new generic function for ‘rbind’ in package ‘BiocGenerics’
Creating a new generic function for ‘do.call’ in package ‘BiocGenerics’
Creating a new generic function for ‘duplicated’ in package ‘BiocGenerics’
Creating a new generic function for ‘anyDuplicated’ in package ‘BiocGenerics’
Creating a new generic function for ‘eval’ in package ‘BiocGenerics’
Creating a new generic function for ‘pmax’ in package ‘BiocGenerics’
Creating a new generic function for ‘pmin’ in package ‘BiocGenerics’
...
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/RforProteomics/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -c Spectrum1_class.c -o Spectrum1_class.o
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/RforProteomics/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -c Spectrum2_class.c -o Spectrum2_class.o
g++ -std=gnu++14 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/RforProteomics/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -c imp_neighbour_avg.cpp -o imp_neighbour_avg.o
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/RforProteomics/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -c utils.c -o utils.o
g++ -std=gnu++14 -shared -L/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/lib -L/usr/local/lib64 -o MSnbase.so RcppExports.o Spectrum1_class.o Spectrum2_class.o imp_neighbour_avg.o utils.o -L/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/lib -lR
Creating a new generic function for ‘trimws’ in package ‘MSnbase’
in method for ‘coerce’ with signature ‘"MSnSet","SummarizedExperiment"’: no definition for class “SummarizedExperiment”
in method for ‘coerce’ with signature ‘"SummarizedExperiment","MSnSet"’: no definition for class “SummarizedExperiment”
in method for ‘coerce’ with signature ‘"IBSpectra","MSnSet"’: no definition for class “IBSpectra”
in method for ‘plotMzDelta’ with signature ‘"mzRramp"’: no definition for class “mzRramp”


trying URL 'https://bioconductor.org/packages/3.15/bioc/src/contrib/MLInterfaces_1.76.0.tar.gz'
Content type 'application/x-gzip' length 1374688 bytes (1.3 MB)
==================================================
downloaded 1.3 MB

trying URL 'https://bioconductor.org/packages/3.15/data/experiment/src/contrib/msdata_0.36.0.tar.gz'
Content type 'application/x-gzip' length 447124112 bytes (426.4 MB)
==================================================
downloaded 426.4 MB

...
* DONE (biomaRt)
* installing *binary* package ‘caret’ ...
* DONE (caret)
* installing *source* package ‘DEP’ ...
** using staged installation
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading


```
# singleCellTK

<details>

* Version: 
* GitHub: https://github.com/HenrikBengtsson/R.utils
* Source code: NA
* Number of recursive dependencies: 0

</details>

## Error before installation

### Devel

```
Creating a new generic function for ‘append’ in package ‘BiocGenerics’
Creating a new generic function for ‘as.data.frame’ in package ‘BiocGenerics’
Creating a new generic function for ‘cbind’ in package ‘BiocGenerics’
Creating a new generic function for ‘rbind’ in package ‘BiocGenerics’
Creating a new generic function for ‘do.call’ in package ‘BiocGenerics’
Creating a new generic function for ‘duplicated’ in package ‘BiocGenerics’
Creating a new generic function for ‘anyDuplicated’ in package ‘BiocGenerics’
Creating a new generic function for ‘eval’ in package ‘BiocGenerics’
Creating a new generic function for ‘pmax’ in package ‘BiocGenerics’
Creating a new generic function for ‘pmin’ in package ‘BiocGenerics’
...
Creating a generic function for ‘eapply’ from package ‘base’ in package ‘AnnotationDbi’
Creating a generic function for ‘exists’ from package ‘base’ in package ‘AnnotationDbi’
Creating a generic function for ‘sample’ from package ‘base’ in package ‘AnnotationDbi’
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/singleCellTK/S4Vectors/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/singleCellTK/IRanges/include' -I/usr/local/include   -fpic  -g -O2  -c IRanges_stubs.c -o IRanges_stubs.o
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/singleCellTK/S4Vectors/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/singleCellTK/IRanges/include' -I/usr/local/include   -fpic  -g -O2  -c R_init_GenomicAlignments.c -o R_init_GenomicAlignments.o
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/singleCellTK/S4Vectors/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/singleCellTK/IRanges/include' -I/usr/local/include   -fpic  -g -O2  -c S4Vectors_stubs.c -o S4Vectors_stubs.o
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/singleCellTK/S4Vectors/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/singleCellTK/IRanges/include' -I/usr/local/include   -fpic  -g -O2  -c cigar_utils.c -o cigar_utils.o
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/singleCellTK/S4Vectors/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/singleCellTK/IRanges/include' -I/usr/local/include   -fpic  -g -O2  -c coordinate_mapping_methods.c -o coordinate_mapping_methods.o
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/singleCellTK/S4Vectors/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/singleCellTK/IRanges/include' -I/usr/local/include   -fpic  -g -O2  -c encodeOverlaps_methods.c -o encodeOverlaps_methods.o
gcc -shared -L/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/lib -L/usr/local/lib64 -o GenomicAlignments.so IRanges_stubs.o R_init_GenomicAlignments.o S4Vectors_stubs.o cigar_utils.o coordinate_mapping_methods.o encodeOverlaps_methods.o -L/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/lib -lR


trying URL 'https://bioconductor.org/packages/3.15/data/experiment/src/contrib/celldex_1.6.0.tar.gz'
Content type 'application/x-gzip' length 390349 bytes (381 KB)
==================================================
downloaded 381 KB

trying URL 'https://bioconductor.org/packages/3.15/data/experiment/src/contrib/scRNAseq_2.10.0.tar.gz'
Content type 'application/x-gzip' length 349138 bytes (340 KB)
==================================================
downloaded 340 KB

...
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (annotate)


```
### CRAN

```
Creating a new generic function for ‘append’ in package ‘BiocGenerics’
Creating a new generic function for ‘as.data.frame’ in package ‘BiocGenerics’
Creating a new generic function for ‘cbind’ in package ‘BiocGenerics’
Creating a new generic function for ‘rbind’ in package ‘BiocGenerics’
Creating a new generic function for ‘do.call’ in package ‘BiocGenerics’
Creating a new generic function for ‘duplicated’ in package ‘BiocGenerics’
Creating a new generic function for ‘anyDuplicated’ in package ‘BiocGenerics’
Creating a new generic function for ‘eval’ in package ‘BiocGenerics’
Creating a new generic function for ‘pmax’ in package ‘BiocGenerics’
Creating a new generic function for ‘pmin’ in package ‘BiocGenerics’
...
Creating a generic function for ‘eapply’ from package ‘base’ in package ‘AnnotationDbi’
Creating a generic function for ‘exists’ from package ‘base’ in package ‘AnnotationDbi’
Creating a generic function for ‘sample’ from package ‘base’ in package ‘AnnotationDbi’
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/singleCellTK/S4Vectors/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/singleCellTK/IRanges/include' -I/usr/local/include   -fpic  -g -O2  -c IRanges_stubs.c -o IRanges_stubs.o
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/singleCellTK/S4Vectors/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/singleCellTK/IRanges/include' -I/usr/local/include   -fpic  -g -O2  -c R_init_GenomicAlignments.c -o R_init_GenomicAlignments.o
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/singleCellTK/S4Vectors/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/singleCellTK/IRanges/include' -I/usr/local/include   -fpic  -g -O2  -c S4Vectors_stubs.c -o S4Vectors_stubs.o
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/singleCellTK/S4Vectors/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/singleCellTK/IRanges/include' -I/usr/local/include   -fpic  -g -O2  -c cigar_utils.c -o cigar_utils.o
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/singleCellTK/S4Vectors/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/singleCellTK/IRanges/include' -I/usr/local/include   -fpic  -g -O2  -c coordinate_mapping_methods.c -o coordinate_mapping_methods.o
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/singleCellTK/S4Vectors/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/singleCellTK/IRanges/include' -I/usr/local/include   -fpic  -g -O2  -c encodeOverlaps_methods.c -o encodeOverlaps_methods.o
gcc -shared -L/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/lib -L/usr/local/lib64 -o GenomicAlignments.so IRanges_stubs.o R_init_GenomicAlignments.o S4Vectors_stubs.o cigar_utils.o coordinate_mapping_methods.o encodeOverlaps_methods.o -L/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/lib -lR


trying URL 'https://bioconductor.org/packages/3.15/data/experiment/src/contrib/celldex_1.6.0.tar.gz'
Content type 'application/x-gzip' length 390349 bytes (381 KB)
==================================================
downloaded 381 KB

trying URL 'https://bioconductor.org/packages/3.15/data/experiment/src/contrib/scRNAseq_2.10.0.tar.gz'
Content type 'application/x-gzip' length 349138 bytes (340 KB)
==================================================
downloaded 340 KB

...
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (annotate)


```
# TCGAbiolinks

<details>

* Version: 
* GitHub: https://github.com/HenrikBengtsson/R.utils
* Source code: NA
* Number of recursive dependencies: 0

</details>

## Error before installation

### Devel

```
Creating a new generic function for ‘append’ in package ‘BiocGenerics’
Creating a new generic function for ‘as.data.frame’ in package ‘BiocGenerics’
Creating a new generic function for ‘cbind’ in package ‘BiocGenerics’
Creating a new generic function for ‘rbind’ in package ‘BiocGenerics’
Creating a new generic function for ‘do.call’ in package ‘BiocGenerics’
Creating a new generic function for ‘duplicated’ in package ‘BiocGenerics’
Creating a new generic function for ‘anyDuplicated’ in package ‘BiocGenerics’
Creating a new generic function for ‘eval’ in package ‘BiocGenerics’
Creating a new generic function for ‘pmax’ in package ‘BiocGenerics’
Creating a new generic function for ‘pmin’ in package ‘BiocGenerics’
...
  rgeos: versions of GEOS runtime 3.9.1-CAPI-1.14.2
and GEOS at installation 3.4.2-CAPI-1.8.2differ
g++ -std=gnu++14 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/TCGAbiolinks/Rcpp/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/TCGAbiolinks/RcppEigen/include' -I/usr/local/include  -DNDEBUG -DEIGEN_DONT_VECTORIZE -fpic  -g -O2  -c external.cpp -o external.o
g++ -std=gnu++14 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/TCGAbiolinks/Rcpp/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/TCGAbiolinks/RcppEigen/include' -I/usr/local/include  -DNDEBUG -DEIGEN_DONT_VECTORIZE -fpic  -g -O2  -c glmFamily.cpp -o glmFamily.o
g++ -std=gnu++14 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/TCGAbiolinks/Rcpp/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/TCGAbiolinks/RcppEigen/include' -I/usr/local/include  -DNDEBUG -DEIGEN_DONT_VECTORIZE -fpic  -g -O2  -c optimizer.cpp -o optimizer.o
g++ -std=gnu++14 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/TCGAbiolinks/Rcpp/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/TCGAbiolinks/RcppEigen/include' -I/usr/local/include  -DNDEBUG -DEIGEN_DONT_VECTORIZE -fpic  -g -O2  -c predModule.cpp -o predModule.o
g++ -std=gnu++14 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/TCGAbiolinks/Rcpp/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/TCGAbiolinks/RcppEigen/include' -I/usr/local/include  -DNDEBUG -DEIGEN_DONT_VECTORIZE -fpic  -g -O2  -c respModule.cpp -o respModule.o
g++ -std=gnu++14 -shared -L/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/lib -L/usr/local/lib64 -o lme4.so external.o glmFamily.o optimizer.o predModule.o respModule.o -L/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/lib -lR
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG   -I/usr/local/include   -fpic  -g -O2  -c sva.c -o sva.o
gcc -shared -L/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/lib -L/usr/local/lib64 -o sva.so sva.o -L/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/lib -lR


trying URL 'https://bioconductor.org/packages/3.15/bioc/src/contrib/minet_3.54.0.tar.gz'
Content type 'application/x-gzip' length 51301 bytes (50 KB)
==================================================
downloaded 50 KB

trying URL 'https://bioconductor.org/packages/3.15/bioc/src/contrib/pathview_1.36.0.tar.gz'
Content type 'application/x-gzip' length 2216624 bytes (2.1 MB)
==================================================
downloaded 2.1 MB

...
* DONE (sesame)
* installing *binary* package ‘rstatix’ ...
* DONE (rstatix)
* installing *binary* package ‘ggpubr’ ...
* DONE (ggpubr)
* installing *binary* package ‘survminer’ ...
* DONE (survminer)

The downloaded source packages are in
	‘/scratch/henrik/641569/RtmpSeLtR5/downloaded_packages’


```
### CRAN

```
Creating a new generic function for ‘append’ in package ‘BiocGenerics’
Creating a new generic function for ‘as.data.frame’ in package ‘BiocGenerics’
Creating a new generic function for ‘cbind’ in package ‘BiocGenerics’
Creating a new generic function for ‘rbind’ in package ‘BiocGenerics’
Creating a new generic function for ‘do.call’ in package ‘BiocGenerics’
Creating a new generic function for ‘duplicated’ in package ‘BiocGenerics’
Creating a new generic function for ‘anyDuplicated’ in package ‘BiocGenerics’
Creating a new generic function for ‘eval’ in package ‘BiocGenerics’
Creating a new generic function for ‘pmax’ in package ‘BiocGenerics’
Creating a new generic function for ‘pmin’ in package ‘BiocGenerics’
...
  rgeos: versions of GEOS runtime 3.9.1-CAPI-1.14.2
and GEOS at installation 3.4.2-CAPI-1.8.2differ
g++ -std=gnu++14 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/TCGAbiolinks/Rcpp/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/TCGAbiolinks/RcppEigen/include' -I/usr/local/include  -DNDEBUG -DEIGEN_DONT_VECTORIZE -fpic  -g -O2  -c external.cpp -o external.o
g++ -std=gnu++14 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/TCGAbiolinks/Rcpp/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/TCGAbiolinks/RcppEigen/include' -I/usr/local/include  -DNDEBUG -DEIGEN_DONT_VECTORIZE -fpic  -g -O2  -c glmFamily.cpp -o glmFamily.o
g++ -std=gnu++14 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/TCGAbiolinks/Rcpp/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/TCGAbiolinks/RcppEigen/include' -I/usr/local/include  -DNDEBUG -DEIGEN_DONT_VECTORIZE -fpic  -g -O2  -c optimizer.cpp -o optimizer.o
g++ -std=gnu++14 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/TCGAbiolinks/Rcpp/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/TCGAbiolinks/RcppEigen/include' -I/usr/local/include  -DNDEBUG -DEIGEN_DONT_VECTORIZE -fpic  -g -O2  -c predModule.cpp -o predModule.o
g++ -std=gnu++14 -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG  -I'/c4/home/henrik/repositories/R.utils/revdep/library/TCGAbiolinks/Rcpp/include' -I'/c4/home/henrik/repositories/R.utils/revdep/library/TCGAbiolinks/RcppEigen/include' -I/usr/local/include  -DNDEBUG -DEIGEN_DONT_VECTORIZE -fpic  -g -O2  -c respModule.cpp -o respModule.o
g++ -std=gnu++14 -shared -L/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/lib -L/usr/local/lib64 -o lme4.so external.o glmFamily.o optimizer.o predModule.o respModule.o -L/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/lib -lR
gcc -I"/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/include" -DNDEBUG   -I/usr/local/include   -fpic  -g -O2  -c sva.c -o sva.o
gcc -shared -L/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/lib -L/usr/local/lib64 -o sva.so sva.o -L/software/c4/cbi/software/R-4.2.0-gcc10/lib64/R/lib -lR


trying URL 'https://bioconductor.org/packages/3.15/bioc/src/contrib/minet_3.54.0.tar.gz'
Content type 'application/x-gzip' length 51301 bytes (50 KB)
==================================================
downloaded 50 KB

trying URL 'https://bioconductor.org/packages/3.15/bioc/src/contrib/pathview_1.36.0.tar.gz'
Content type 'application/x-gzip' length 2216624 bytes (2.1 MB)
==================================================
downloaded 2.1 MB

...
* DONE (sesame)
* installing *binary* package ‘rstatix’ ...
* DONE (rstatix)
* installing *binary* package ‘ggpubr’ ...
* DONE (ggpubr)
* installing *binary* package ‘survminer’ ...
* DONE (survminer)

The downloaded source packages are in
	‘/scratch/henrik/641569/RtmpSeLtR5/downloaded_packages’


```
